### PR TITLE
earthly 0.6.17

### PR DESCRIPTION
**Pull request by griswoldthecat**
update earthly to 0.6.17

### CODE REVIEW

File: Formula/earthly.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Earthly < Formula
   def install
     # the earthly_gitsha variable is required by the earthly release script, moving this value it into
```
Comment (gromgit) [inline Formula/earthly.rb:21]:
```suggestion
    # the earthly_gitsha variable is required by the earthly release script, moving this value into
```

File: Formula/earthly.rb [hunk 2]
```diff
@@ -18,9 +18,9 @@ class Earthly < Formula
   depends_on "go" => :build
 
   def install
-    # the earthly_gitsha variable is required by the earthly release script, moving this value it into
+    # the earthly_gitsha variable is required by the earthly release script, moving this value into
     # the ldflags string will break the upstream release process.
-    earthly_gitsha = "05fc449487ad0c8a1721b4fbfc527eb9870dfcfd"
+    earthly_gitsha = "7e4f1df4c124db1644d51d312b19313217cbe478"
```
[the remark this answers is not in the document]
Comment (alexcb) [reply Formula/earthly.rb:23]: Can you point us to an example of a formula that does this @SMillerDev ?

**Review by carlocab — DISMISSED** [review]
This works for me, especially since upstream have automation that helps us avoid a Git clone.

**Review by chenrui333 — DISMISSED** [review]
LGTM

File: Formula/earthly.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexcb) [reply Formula/earthly.rb]: @SMillerDev I switched it to perform a `git clone`.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**